tcms/testcases/migrations/0007_convert_is_automated_to_boolean.py

```diff
@@ -16,6 +16,19 @@ def forward_migrate_data(apps, schema_editor):
         tc_history.save()
 
 
+def backward_restore_data(apps, schema_editor):
+    test_case_model = apps.get_model('testcases', 'TestCase')
+    historical_test_case_model = apps.get_model('testcases', 'HistoricalTestCase')
+
+    for test_case in test_case_model.objects.all():
+        test_case.new_is_automated = int(test_case.is_automated)
```

Comment:
this is the other way around. You are taking the value of `new_is_automated` (which is bool) and assigning it to `is_automated` (which is int).